scripts/start.sh

```diff
@@ -28,9 +28,13 @@ _base_/start.sh
 auth/start.sh
 aether/start.sh
 
-if [ "$ENABLE_CONNECT" = true ]; then
+
+if [ "$AETHER_CONNECT_MODE" = 'LOCAL' ]; then
     connect/start.sh
 fi
+if [ "$AETHER_CONNECT_MODE" = 'CONFLUENT' ]; then
+    connect/start_ccloud.sh
+fi
```

Comment:
you are repeating this instruction so much, maybe you should move it to the `connect/start.sh` file and it should decide which environment start or setup or...